core/db/entities.py

```diff
@@ -242,7 +242,7 @@ class Project(BaseModel):
     primary_intervention_theme = sqla.Column(sqla.String(), nullable=False)
     location_multiplicity = sqla.Column(sqla.String, nullable=True)
     locations = sqla.Column(sqla.String, nullable=False)
-    postcodes = sqla.Column(sqla.String, nullable=True)
+    postcodes = sqla.Column(sqla.ARRAY(sqla.String), nullable=True)
```

Comment:
so I pulled the branch and checked the extract behaviour in tests, and our potcode data now in quare brackets (python list) in serialisation tests. like so: 

`['BS3 1AB']`

I'm wondering if:
1) the fact this didn't cause any test failures means we probably don't have specific coverage of this particular field being serialised
2) How does Excel output handle this?

Reply:
Yeah I've tested locally and can confirm that now "postcodes" column in Excel file looks like:

`['SW1A 2AA', 'BT1 1AA']`

When really it should look like it did before, ie just comma separated list, this is from a previous Excel file:

`SW1A 2AA,BT1 1AA`

Reply:
We should probably:
1) Try and fix the extract / serialisation to match the old format on extract (I presume some of Hrishi's E2E tests will flag this failure)
2) add a test assertion on the format of this particular field, probably in serialisation_tests